src/components/side_panel/pivot/pivot_layout_configurator/pivot_dimension/pivot_dimension.ts

```diff
@@ -27,6 +29,22 @@ export class PivotDimension extends Component<Props, SpreadsheetChildEnv> {
   static props = {
     dimension: Object,
     onRemoved: { type: Function, optional: true },
+    onNameUpdated: { type: Function, optional: true },
     slots: { type: Object, optional: true },
   };
+
+  private state: { isUpdate: boolean } = useState({ isUpdate: false });
+
+  setup() {
+    useAutofocus({ refName: "autofocus" });
+  }
+
+  updateName(ev: InputEvent) {
+    const value = (ev.target as HTMLInputElement).value;
+    this.props.onNameUpdated?.(
+      this.props.dimension,
+      value === "" || value.startsWith("=") ? undefined : value
```

Comment:
why `|| value.startsWith("=")`

Reply:
Because if you enter a formula, the result in the pivot will be the formula text. And when you press "Enter", ranges in the "formula" are highlighted :p
So it's the simplier way I found to avoid that